src/Lecture3.hs

```diff
@@ -52,15 +51,16 @@ data Weekday
     | Friday
     | Saturday
     | Sunday
-    deriving (Show, Eq)
+    deriving (Show, Eq, Enum, Bounded, Ord)
 
 {- | Write a function that will display only the first three letters
 of a weekday.
 
 >>> toShortString Monday
 "Mon"
 -}
-toShortString = error "TODO"
+toShortString :: Weekday -> String
+toShortString x = take 3 (show x)
```

Comment:
Absolutely correct implementation 👍🏻 
You can use function composition and eta-reduce it:
```suggestion
toShortString = take 3 . show
```